CNAME

```diff
@@ -1 +0,0 @@
-errorprone.info
```

Comment:
I think we do need the CNAME file, based on the revision history of this file.  (I know nothing more than that, but it looks like Alex tried deleting it before, and then had to revert the change)